docs/src/installation.md

```diff
@@ -31,33 +38,47 @@ https://source.android.com/setup/build/running
     09071JEC217048  device
     ```
 
- 3. First flash your custom AVB signing key using
+ 3. Unlock the bootloader by running
+    ```console
+    $ fastboot flashing unlock
+    ```
+    Select the option to unlock the device and confirm.
```

Comment:
May want to put a warning here that this step does a factory reset, deleting all user data etc.

Reply:
Added, thanks for reading over this!